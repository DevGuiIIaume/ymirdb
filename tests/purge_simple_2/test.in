set a 1
snapshot
set b 2
snapshot
set c 3
snapshot
set d 4
snapshot
set e 5
snapshot
list entries
purge a
list entries
checkout 1
list entries
checkout 2
list entries
checkout 3
list entries
checkout 4
list entries
checkout 5
list entries
purge c
checkout 1
list entries
checkout 2
list entries
checkout 3
list entries
get c
checkout 4
list entries
checkout 5
list entries
rollback 3
list entries
purge b
list entries
list snapshots
bye
