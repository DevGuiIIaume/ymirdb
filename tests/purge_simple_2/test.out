> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> saved as snapshot 3

> ok

> saved as snapshot 4

> ok

> saved as snapshot 5

> e [5]
d [4]
c [3]
b [2]
a [1]

> ok

> e [5]
d [4]
c [3]
b [2]

> ok

> no entries

> ok

> b [2]

> ok

> c [3]
b [2]

> ok

> d [4]
c [3]
b [2]

> ok

> e [5]
d [4]
c [3]
b [2]

> ok

> ok

> no entries

> ok

> b [2]

> ok

> b [2]

> no such key

> ok

> d [4]
b [2]

> ok

> e [5]
d [4]
b [2]

> ok

> b [2]

> ok

> no entries

> 3
2
1

> bye
